import_3dm/converters/__init__.py

```diff
@@ -53,19 +55,18 @@ def convert_object(context, ob, name, layer, rhinomat, scale):
     data = None
 
     if ob.Geometry.ObjectType in RHINO_TYPE_TO_IMPORT:
-        data = RHINO_TYPE_TO_IMPORT[ob.Geometry.ObjectType](context, ob, name, scale)
```

Comment:
this looks a bit confusing here, I had to return the created empty from the converter, to be able to link it to its layer. therefore I needed to change the if statement to check for instance references and treat them differently from geometry objects